omdata/eval_workflows/evaluator.py

```diff
@@ -0,0 +1,839 @@
+from __future__ import annotations
+
+import numpy as np
+from pymatgen.io.ase import MSONAtoms
+from schrodinger.application.jaguar.utils import mmjag_reset_connectivity
+from schrodinger.application.matsci.aseutils import get_structure
+from schrodinger.comparison.atom_mapper import ConnectivityAtomMapper
+from schrodinger.structure import Structure
+from schrodinger.structutils import rmsd
+from scipy.optimize import linear_sum_assignment
+
+boltzmann_constant = 8.617333262 * 10**-5
+
+
+def renumber_molecules_to_match(mol_list):
+    """
+    Ensure that topologically equivalent sites are equivalently numbered
+    """
+    mapper = ConnectivityAtomMapper(use_chirality=False)
+    atlist = range(1, mol_list[0].atom_total + 1)
+    renumbered_mols = [mol_list[0]]
+    for mol in mol_list[1:]:
+        _, r_mol = mapper.reorder_structures(mol_list[0], atlist, mol, atlist)
+        renumbered_mols.append(r_mol)
+    return renumbered_mols
+
+
+def rmsd_wrapper(st1: Structure, st2: Structure) -> float:
+    """
+    Wrapper around Schrodinger's RMSD calculation function.
+    """
+    assert (
+        st1.atom_total == st2.atom_total
+    ), "Structures must have the same number of atoms for RMSD calculation"
+    if st1 == st2:
+        return 0.0
+    at_list = list(range(1, st1.atom_total + 1))
+    return rmsd.superimpose(st1, at_list, st2.copy(), at_list, use_symmetry=True)
+
+
+def sdgr_rmsd(atoms1, atoms2):
+    """
+    Calculate the RMSD between two sets of atoms.
+    """
+    atoms1 = MSONAtoms.from_dict(atoms1)
+    atoms2 = MSONAtoms.from_dict(atoms2)
+
+    st1 = get_structure(atoms1)
+    mmjag_reset_connectivity(st1)
+    st2 = get_structure(atoms2)
+    mmjag_reset_connectivity(st2)
+    renumbered_sts = renumber_molecules_to_match([st1, st2])
```

Comment:
```suggestion
    try:
        renumbered_sts = renumber_molecules_to_match([st1, st2])
    except ConformerError:
        copy_bonding(st1, st2)
```
You'll need these imports `from schrodinger.comparison.exceptions import ConformerError` and `from schrodinger.application.jaguar.autots_bonding import copy_bonding`